scripts/go-function-build.sh

```diff
@@ -2,9 +2,12 @@
 
 set -euo pipefail
 
+source ../../scripts/get-versions.sh
+
 image_tag=${TAG:-dev}
+tags=$(validate_version "${image_tag}")
 prefix="release-go-functions-"
 [[ "${image_tag}" = "${prefix}"* ]] && image_tag="${image_tag#$prefix}"
-image_name=gcr.io/kpt-functions/"${CURRENT_FUNCTION}"
-image="${image_name}":"${image_tag}"
-docker build -t "${image}" -t "${image_name}" -f "${CURRENT_FUNCTION}"/Dockerfile "${CURRENT_FUNCTION}"
+for tag in ${tags}; do
```

Comment:
you probably want to invoke `docker build` once and then use `docker tag` instead of building N times.